src/main/java/org/spongepowered/common/mixin/core/item/MixinItemFishingRod.java

```diff
@@ -54,10 +54,12 @@ public ItemStack onItemRightClick(ItemStack itemStack, World world, EntityPlayer
             player.fishEntity.handleHookRetraction();
         } else {
             EntityFishHook fishHook = new EntityFishHook(world, player);
-            if (!Sponge.getGame().getEventManager()
+            if (world.isRemote || !Sponge.getGame().getEventManager()
```

Comment:
@JBYoshi 

They'll have to learn.

@ryantheleach 

Way too messy. Almost all events would have that marker. If anything, I'd love @Subscribe to be changed to allow you to target a specific platform; as in your handler wouldn't be in the server or client's handlers list at all. Kinda how @SideOnly behaves in FML.